src/Atmos/Model/tendencies_moisture.jl

```diff
@@ -47,6 +47,11 @@ function flux(::MoistureDiffusion{IceMoisture}, atmos, args)
     return d_q_ice * state.ρ
 end
 
+function flux(::HyperdiffViscousFlux{TotalMoisture}, atmos, args)
+    @unpack state, hyperdiffusive = args
+    return hyperdiffusive.hyperdiffusion.ν∇³q_tot * state.ρ
```

Comment:
Should this also work with nonequilibrium moisture model? Then we would also need this for cloud liquid and ice variables? How about precipitation variables?

Reply:
Yep, absolutely. We can add `flux(::HyperdiffViscousFlux{LiquidMoisture}` and `flux(::HyperdiffViscousFlux{IceMoisture}` methods if that's needed in the future.

Reply:
@jiahe23 - Should we do it now? Or is it obvious to everyone that it should not be used with moisture and precipitation vars right now?

Reply:
For now, we don't have a hyperdiffussion written for nonequil. We only have these two options
- [DryBiharmonic](https://github.com/CliMA/ClimateMachine.jl/blob/71c045457c177c0c1f2cfbc9008162fa52b87435/src/Common/TurbulenceClosures/TurbulenceClosures.jl#L892) 
- [EquilMoistBiharmonic](https://github.com/CliMA/ClimateMachine.jl/blob/71c045457c177c0c1f2cfbc9008162fa52b87435/src/Common/TurbulenceClosures/TurbulenceClosures.jl#L804)

How about leaving the non-equil related var out for now?